src/i18n/README.md

```diff
@@ -0,0 +1,3 @@
+### Translations
+You can translate Hunger Games at https://crowdin.com/translate/openfoodfacts/1942. Please do not edit files directly, as translations made on GitHub will be overwritten.
+If you have added or modified a string, you should update the crowdin-trigger branch from master. This will trigger import within Crowdin, and 1 hour later, it will create a new pull request with added or modified strings in all language files, plus all added or modified translations.
```

Comment:
> you should update the crowdin-trigger branch from master

How do you do that? Is it the following command?
```bash
git rebase crowdin-trigger master
```

Reply:
I use the GitHub Desktop update menu usually (except this week where I'm stuck on a Chromebook without any access to commandline). A rebase should theoretically do it, and it's probably also possible to rebase from the web interface.

Reply:
@alexfauquette translations are now refreshed automatically every day